debian/salsa-ci.yml

```diff
@@ -103,24 +102,18 @@ blhc:
   # Prime the apt cache so later apt commands can run
   apt-get update
 
-# Readline was removed from Debian Sid (and Bullseye) in Feb 2021. To be able to install older
-# versions of MariaDB that depend on it, fetch and install it from Buster.
-.test-install-readline-in-sid-for-backwards-compat: &test-install-readline-in-sid-for-backwards-compat |
-  curl -O http://ftp.de.debian.org/debian/pool/main/r/readline5/libreadline5_5.2+dfsg-3+b13_amd64.deb
-  apt install -y ./libreadline5_5.2+dfsg-3+b13_amd64.deb
-
 .test-verify-initial: &test-verify-initial |
   dpkg -l | grep -iE 'maria|mysql|galera' || true # List installed
   service mysql status || service mariadb status # Early MariaDB 10.5 only had 'mariadb'
   mysql --skip-column-names -e "select @@version, @@version_comment" # Show version
-  mysql --table -e 'SHOW DATABASES;' # List databases before upgrade
+  mysql --table -e "SHOW DATABASES;" # List databases before upgrade
   mysql --table -e "SELECT host,user,plugin,authentication_string FROM user;" mysql
   mysql --table -e "SELECT * FROM plugin;" mysql
   mysql --table -e "SHOW PLUGINS;" mysql
 
-.test-enable-sid-repos: &test-enable-sid-repos |
-  # Replace any old repos with just Sid
-  echo 'deb http://deb.debian.org/debian sid main' > /etc/apt/sources.list
+.test-enable-bullseye-repos: &test-enable-bullseye-repos |
+  # Replace any old repos with just Bullseye
+  echo 'deb http://deb.debian.org/debian bullseye main' > /etc/apt/sources.list
```

Comment:
Please use '"' as everything else is converted to use it

Reply:
Should this use ${RELEASE} not bullseye?

Reply:
Using now `"` everywhere in deb lines. In the repo change `${RELEASE}` can't be used, as it needs to control exactly what distro release the upgrade test is going into.